HKMP/Game/Client/Entity/EntityManager.cs

```diff
@@ -273,6 +293,53 @@ byte state
             entity.UpdateState(state);
         }
 
+        /**
+         * Initializes an entity with a state if we have one for it.
+         */
+        private bool InitializeEntity(
```

Comment:
Return type of this method is unused. If it is isn't needed in the future, it can be removed.
```suggestion
        private void InitializeEntity(
```